sources/packages/web/src/views/aest/student/applicationDetails/RestrictionsManagement.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <full-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Application history"
```

Comment:
`title="Student Applications"`

Reply:
Updated.

Reply:
`Application history` was the latest content from assessments page. But I would consider it not a blocker as we do not precisely know what is the content with respect to the ticket.